packages/next/server/api-utils/node.ts

```diff
@@ -510,19 +511,23 @@ function setPreviewData<T>(
       httpOnly: true,
       sameSite: process.env.NODE_ENV !== 'development' ? 'none' : 'lax',
       secure: process.env.NODE_ENV !== 'development',
-      path: '/',
       ...(options.maxAge !== undefined
         ? ({ maxAge: options.maxAge } as CookieSerializeOptions)
         : undefined),
+      ...(options.path !== undefined
+        ? ({ path: options.path } as CookieSerializeOptions)
```

Comment:
while checking `CookieSerializeOptions` type definitions I noticed that it'd be good to update it to v0.5.1 to get the latest (and more complete definitions). `/@types/cookie/0.5.1` has correct links to https://datatracker.ietf.org/doc/html/rfc6265#section-5.2.4 which is way more helpful to understand how this `path` and `maxAge` behave